pkg/billing/client.go

```diff
@@ -1,21 +1,10 @@
 package billing
```

Comment:
Does it make sense to get rid of this package and move all the stuff to either agent/billing or reporting?

Reply:
I'd like to do that, yeah. Wanted to leave it outside the scope of this change for now though — will keep this comment open as a reminder.

Reply:
⇒ #1139